openassessment/xblock/staff_area_mixin.py

```diff
@@ -266,6 +266,12 @@ def get_student_submission_context(self, student_username, submission):
         if submission:
             context["file_upload_type"] = self.file_upload_type
             context["staff_file_urls"] = self.get_download_urls_from_submission(submission)
+            if self.file_upload_response and not any(context['staff_file_urls']):
+                logger.info(u"Checking student module for upload info for user: {username} in block: {block}".format(
```

Comment:
What do you think a log after calling `self.get_files_info_from_user_state(student_username)` where you cal also log if the state was restored from CSM service?

Reply:
Do you mean checking before & after CSM call and logging both data values?

Reply:
not necessarily logging the data value, but just a flag that files were missing and were found / not found from csm; or something like that.